nsv13/code/modules/overmap/knpc.dm

```diff
@@ -354,14 +354,14 @@ of a specific action goes up, to encourage skynet to go for that one instead.
 			if(get_dist(H, OM) > HA.view_range || !can_see(H, OM, HA.view_range))
 				continue
```

Comment:
Can you just change this global mechas list access to be 
for(var/obj/mecha/OM in guessed_objects)
with an if(!ismecha(OM)) check or something
Please 🥺 (You will get a treat)